src/types.ts

```diff
@@ -104,11 +105,29 @@ export interface RunmeTaskDefinition {
   cwd?: string
 }
 
-export interface NotebookCellAnnotations {
-  'runme.dev/uuid': string
-  background: boolean
-  interactive: boolean
-  closeTerminalOnSuccess: boolean
-  mimeType: string
-  name: string
-}
+const falseyBoolean = z.preprocess((subject) => {
+  if (typeof subject === 'string' && subject.toLowerCase() === 'false') {
+    return false
+  }
+  return Boolean(subject)
+}, z.boolean())
+
+export const CellAnnotationsSchema = z.object({
+  'runme.dev/uuid': z.string().optional(),
```

Comment:
I'ld suggest to move this into `constants.ts` as this is rather an object initiation than a type definition

Reply:
Totally. Or if we'll even have more schemas we could just move them to a `schemas.ts`.

Reply:
Moved all the zod schemas to schemas.ts